src/database/model/QueueEntry.ts

```diff
@@ -27,6 +27,9 @@ export class QueueEntry {
 	@Column()
 	isDone: boolean;
 
+	@Column()
+	likeCount: number;
```

Comment:
How about nowPlayingCalls for now? I'll try to add proper user ID tracking in a later version and then we can bring this name back.

Reply:
I'd like to nail as close to the final implementation of this feature as we can, to avoid confusing our DJ as he's likely to take into account the exaggerated stats from this first go. I'll ask him about it to see whether he wants to allow only each user's first call through to the "like" counter.

What about `haveCalledNowPlaying`, using the ["simple-array"](https://typeorm.io/#/entities/simple-array-column-type) column type? If I understand correctly, this should only be a minor change from "increment the number on this column" to "insert this user ID to the array on this column, if the user ID is unique."

Reply:
Sounds good, I'll code it up ASAP

Reply:
My main concern is that database migrations are hard 😅.  I'd like to nail down the final column name and type soon early on, to avoid having to migrate or nuke the database later, even if that means iterating on the design in the PR stage.

Reply:
Another option would be to use `nowPlayingCalls`, merge this work with that column name, and not publish a new version until we're satisfied with the behavior of this feature.